docs/installation.rst

```diff
@@ -1379,8 +1379,14 @@ Prior to SIP-15 SQLAlchemy used inclusive endpoints however these may behave lik
 To remedy this rather than having to define the date/time format for every non-IS0 8601 date-time column, once can define a default column mapping on a per database level via the ``extra`` parameter ::
 
     {
+        "main_dttm_column": "ds",
```

Comment:
I wonder if this should be named `main_dttm_col` to be consistent with the datasource level field.

Reply:
I sense these default settings (including the `python_date_format_by_column_name` should be documented outside of the SIP-15 section (this section can then reference said section). This would help to provide context regarding default database specific fields which may be inherited either by new Superset tables going forward or used as a fallback (in the case of SIP-15) if a value isn't defined.

Additionally I was wondering if having all these nested under a `defaults` section ensures more consistency (and thus no need for the `default_` prefix), 

```json
{
    "defaults": {
        "dttm_column_names": ["ds", "hour_ts"],
        "main_dttm_col": "ds",
        "expression_by_column_name": {
            "hour_ts": "CAST(hour_ts as INTEGER)"
        },
        "python_date_format_by_column_name": {
            "ds": "%Y-%m-%d",
            "hour_ts": "epoch_s"
        }	       
    }
}
```

would be better documented in their own section, i.e., outside of SIP-15. It could then be clearer to the reader that these database level overrides